corehq/apps/sms/models.py

```diff
@@ -235,6 +238,125 @@ def __unicode__(self):
         to_from = (self.direction == INCOMING) and "from" or "to"
         return "%s (%s %s)" % (str, to_from, self.phone_number)
 
+    @classmethod
+    def _migration_get_fields(cls):
+        return [field for field in SMS._migration_get_fields() if not field.startswith('fri_')]
```

Comment:
Trying to syncing an SMSLog with the fri_ fields will actually make getattr() fail because those aren't attributes of an SMSLog.